frontend/src/components/transactions/ActionMessage.tsx

```diff
@@ -7,6 +7,7 @@ export interface Props<A> {
   actionKind: keyof TransactionMessageRenderers;
   actionArgs: A;
   transaction: T.Transaction;
+  showArg?: boolean;
```

Comment:
This way you leak the abstraction. Let's rename it to a more general thing `showDetails`.